test/AlarmClockTest.cpp

```diff
@@ -9,17 +9,15 @@
 #include "StopWatch.h"
 #include <chrono>
 
-// The timing leeway was determined
-// experimentally. It takes around 130-160 
-// microseconds on average to get through
-// one while loop of 500ms sleep. 
-int kTimingLeeway = 250;
+unsigned int AlarmClockTest::kFakeSleepUs = 0;
```

Comment:
This is the one I meant. Should it be an atomic? It looks like you have synchronous access, but it is accessed from multiple threads.